src/app/components/index.jsx

```diff
@@ -26,6 +39,21 @@ class App extends Component {
           {/* Below line proves that the client talks to the api */}
           {/* Should output: Hello from the Above Curve API! */}
           {this.state.data.message}
+
+          {/* This is how to call a custom Google Analytics event */}
+          {/* This button sends a "TEST" event to GA when the button is clicked */}
+          <button
+            onClick={() => {
+              Event(
+                "TEST",
+                "Someone clicked on the Google Analytics test button.",
+                "LANDING_PAGE"
+              );
+              console.log("clicked!");
+            }}
+          >
+            Test Google Analytics event tracking.
+          </button>
```

Comment:
button fires and console.log registers.  Is there some way for me to see the results in GA?

Reply:
I will look into adding team members tomorrow, there are some tricks to this that I'm not 100% sure on yet. I have not got this to work on localhost even though I have a tracking code for local, this does work in Dev but only if I open it in incognito (probably ip tracks me and won't display self hits).